modules/doi/includes/utilities.inc

```diff
@@ -88,7 +98,38 @@ function islandora_doi_get_doi_name_from_url($doi_url) {
 }
 
 /**
- * Perform a request to a metadata provider for the given ID.
```

Comment:
@bryjbrown [This deleted comment line](https://github.com/Islandora/islandora_scholar/pull/285/files#diff-52bb712f031b7c5151fc17f7508f7213L91) should actually replace [this newly inserted comment line](https://github.com/Islandora/islandora_scholar/pull/285/files#diff-52bb712f031b7c5151fc17f7508f7213R132) (now, CrossRef is potentially only one of many metadata providers). It seems that the update got reverted during merging of Islandora#280.
(Recap: `s/CrossRef/a metadata provider/` on [L132](https://github.com/bryjbrown/islandora_scholar/blob/cef53684998ed85ffdf02378236cefda942dc8fb/modules/doi/includes/utilities.inc#L132) of `modules/doi/includes/utilities.inc`)

Reply:
:+1: